django_project/stripe_payment/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import datetime
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('certification', '0019_auto_20180420_1205'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TopupPayment',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('payment_id', models.CharField(max_length=32)),
+                ('time_transaction', models.DateTimeField(default=datetime.datetime.now)),
+                ('amount', models.FloatField(help_text='This is amount of dollar user paid.')),
+                ('description', models.CharField(max_length=32, null=True, blank=True)),
+                ('certifying_organisation', models.ForeignKey(to='certification.CertifyingOrganisation')),
```

Comment:
We want the payments to be usable in different ways on the site:

* certificates
* sponsors
* donations
* premium subscription (e.g. use own DNS)
* bug bounties
* other goods still to be thought of

Can we architect things so that the payment does not link to a specific model?

Reply:
yes, we can @timlinux , maybe with 
`id` & `model name` field?